ocs_ci/ocs/resources/pvc.py

```diff
@@ -173,3 +173,16 @@ def get_all_pvcs(namespace=None):
     )
     out = ocp_pvc_obj.get()
     return out
+
+
+def get_pvc_access_mode(pvc_obj):
```

Comment:
I think it would be better to have this as a method in the PVC class

Reply:
Will update the code  accordingly